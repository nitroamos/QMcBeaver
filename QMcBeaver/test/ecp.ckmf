# Optimization tag written Thu Feb 12 18:31:59 2009
#  iteration            Eavg            Estd    Num. Samples           Tcorr      Acceptance         <p dR2>     Sample Var.         Summary
#       9952   -4.9382138357   1.8212649e-03           10897   15.6995573065    0.992244(44)     0.35914(12)    0.0023021093     -4.9382(18) 
# Optimization tag written Wed Feb 11 22:58:16 2009
#  iteration            Eavg            Estd    Num. Samples           Tcorr      Acceptance         <p dR2>     Sample Var.         Summary
#      50000   -4.9388788120   6.7494766e-04           93255   15.8409766689    0.992297(15)    0.358827(39)    0.0026817958    -4.93888(67) 
# Optimization tag written Tue Feb 10 22:14:08 2009
#  iteration            Eavg            Estd    Num. Samples           Tcorr      Acceptance         <p dR2>     Sample Var.         Summary
#     100000   -5.7831964649   3.1533073e-03          195190   27.4240386879    0.986586(15)    0.356781(27)    0.0707711951     -5.7832(32) 
# Optimization tag written Tue Feb 10 10:31:28 2009
#  iteration            Eavg            Estd    Num. Samples           Tcorr      Acceptance         <p dR2>     Sample Var.         Summary
#      61603   -5.7228218207   5.3031347e-03          130023   50.4082759313    0.994172(14)    0.179483(17)    0.0725404626     -5.7228(53) 
# Created on Tue, 10 Feb 2009 01:01:01
# Using gamess output file:  /net/hulk/home5/amosa/ckmf_origs/SiH2/pseudo/p3b1.log
# Using ckmft template file: /ul/amosa/ckmf_origs/optimize.ckmft
# Orbitals are: --- OPEN SHELL ORBITALS --- GENERATED AT Mon Feb  9 17:00:24 2009
# This file created by Jaguar v7.5                                                
# FINAL ROHF ENERGY IS       -4.8490281676 AFTER   9 ITERATIONS

&flags
# Parameters for QMC
run_type
 variational
dt
 0.02
dt_equilibration
 0.02
number_of_walkers
 2
max_time_steps
 400
equilibration_steps
 50
desired_convergence
 0
iseed
 -1000
optimize_Psi
 0
max_time
 -1
one_e_per_iter
 0
output_interval
 5

# Parameters for wavefunction optimization
optimize_UD_Jastrows
 1
optimize_UU_Jastrows
 1
optimize_DD_Jastrows
 1
optimize_EN_Jastrows
 1
optimize_NEE_Jastrows
 0
optimize_L
 0
optimize_CI
 0
optimize_Orbitals
 0
optimize_Psi_method
 automatic
optimize_Psi_criteria
 generalized_eigenvector
a_diag
 -1e-05
ksi
 0.5
max_optimize_Psi_steps
 30
equilibrate_first_opt_step
 1
equilibrate_every_opt_step
 1
optimization_max_iterations
 1
optimization_error_tolerance
 0.001
singularity_penalty_function_parameter
 1e-06
optimize_Psi_barrier_parameter
 1
numerical_derivative_surface
 umrigar88
line_search_step_length
 Linearize
ck_genetic_algorithm_1_population_size
 1000
ck_genetic_algorithm_1_mutation_rate
 0.2
ck_genetic_algorithm_1_initial_distribution_deviation
 1

# Parameters specific to the Green's function
sampling_method
 umrigar93_importance_sampling
QF_modification_type
 umrigar93_unequalelectrons
umrigar93_equalelectrons_parameter
 0.5
warn_verbosity
 0
rel_cutoff
 100
limit_branching
 1
energy_modification_type
 umrigar93
energy_cutoff_type
 umrigar93
lock_trial_energy
 0
synchronize_dmc_ensemble
 0
synchronize_dmc_ensemble_interval
 1000

# Parameters specific to weights, branching, and fusion
walker_reweighting_method
 umrigar93_probability_weighted
branching_method
 nonunit_weight_branching
branching_threshold
 2
fusion_threshold
 0.45
population_control_parameter
 1
correct_population_size_bias
 1
old_walker_acceptance_parameter
 50

# Parameters for initialization
use_equilibration_array
 0
equilibration_function
 ramp
CKAnnealingEquilibration1_parameter
 500
walker_initialization_method
 dans_walker_initialization
walker_initialization_combinations
 3

# Parameters for added functionality/improvements
calculate_bf_density
 0
use_hf_potential
 0
hf_num_average
 100
replace_electron_nucleus_cusps
 1
print_replacement_orbitals
 0
nuclear_derivatives
 none
future_walking
 0

# Parameters relating to output
checkpoint
 0
checkpoint_interval
 100000
use_available_checkpoints
 0
checkpoint_input_name
 ptz0p0_3b1_1
zero_out_checkpoint_statistics
 1
checkpoint_energy_only
 0
print_configs
 0
print_config_frequency
 50
temp_dir
 /temp1/amosa/ptz0p0_3b1_1
write_all_energies_out
 0
write_electron_densities
 0
max_pair_distance
 -1
print_transient_properties
 0
print_transient_properties_interval
 10000

# Parameters for computation/MPI
parallelization_method
 manager_worker
mpireduce_interval
 100
mpipoll_interval
 5
walkers_per_pass
 1
use_basis_function_interpolation
 0
number_basis_function_interpolation_grid_points
 1000
basis_function_interpolation_first_point
 1e-10

# Parameters for the molecule and wavefunction
atoms
 3
charge
 0
energy
 -4.8490281676
trial_function_type
 restricted
pseudo_gridLevel
 1
pseudo_cutoff
 0.0001
norbitals
 4
nbasisfunc
 64
ndeterminants
 1
link_Jastrow_parameters
 1
link_NEE_Jastrows
 2
link_Orbital_parameters
 1
link_Determinant_parameters
 1
reproduce_NE_with_NEE_jastrow
 1
reproduce_EE_with_NEE_jastrow
 1

# Other parameters
chip_and_mike_are_cool
 Yea_Baby!
& 
&geometry
Si   14     0                     0                    -0.098403            
H    1      0                     2.451377              1.365812            
H    1      0                    -2.451377              1.365812            
&
&pseudopotential
Si-ECP GEN 10 2
    3
                   4    1          1.80721061
          7.22884246    3          9.99633089
         -13.0672559    2          2.50043232
    1
         21.20531613    2          2.26686403
    1
         15.43693603    2          2.11659661
H--ECP NONE
H--ECP NONE
&
&basis
Si1   34    9
  9 s    
           29.724387    0.000607882408032   
           13.681707   -0.000735162436831   
            6.297493   -0.000300325167223   
            2.898645    0.0505396955757     
            1.334205   -0.237323423528      
            0.614115   -0.00114804629034    
            0.282668    0.128279326139      
            0.130108    0.0822239326522     
            0.059887    0.014451181084      
  1 s    
            0.090113    0.117219719667      
  1 s    
            0.507467    0.42851491886       
  9 px   
           13.022097   -0.000775735963917   
            6.246966   -0.00364608072482    
            2.996797    0.00418152483046    
            1.437625   -0.126343183366      
            0.689658    0.0625979885086     
            0.330843    0.100743584635      
            0.158712    0.0596544287507     
            0.076137    0.0175772937007     
            0.036525    0.00179262363351    
  9 py   
           13.022097   -0.000775735963917   
            6.246966   -0.00364608072482    
            2.996797    0.00418152483046    
            1.437625   -0.126343183366      
            0.689658    0.0625979885086     
            0.330843    0.100743584635      
            0.158712    0.0596544287507     
            0.076137    0.0175772937007     
            0.036525    0.00179262363351    
  9 pz   
           13.022097   -0.000775735963917   
            6.246966   -0.00364608072482    
            2.996797    0.00418152483046    
            1.437625   -0.126343183366      
            0.689658    0.0625979885086     
            0.330843    0.100743584635      
            0.158712    0.0596544287507     
            0.076137    0.0175772937007     
            0.036525    0.00179262363351    
  1 px   
            0.056148    0.0389590135516     
  1 py   
            0.056148    0.0389590135516     
  1 pz   
            0.056148    0.0389590135516     
  1 px   
            0.146758    0.12947690772       
  1 py   
            0.146758    0.12947690772       
  1 pz   
            0.146758    0.12947690772       
  1 dxx  
            0.170395    0.0743804734122     
  1 dyy  
            0.170395    0.0743804734122     
  1 dzz  
            0.170395    0.0743804734122     
  1 dxy  
            0.170395    0.128830759041      
  1 dxz  
            0.170395    0.128830759041      
  1 dyz  
            0.170395    0.128830759041      
  1 dxx  
            0.539756    0.559441969216      
  1 dyy  
            0.539756    0.559441969216      
  1 dzz  
            0.539756    0.559441969216      
  1 dxy  
            0.539756    0.968981914568      
  1 dxz  
            0.539756    0.968981914568      
  1 dyz  
            0.539756    0.968981914568      
  1 fxxx 
            0.352999    0.141398489025      
  1 fyyy 
            0.352999    0.141398489025      
  1 fzzz 
            0.352999    0.141398489025      
  1 fxxy 
            0.352999    0.316176633376      
  1 fxxz 
            0.352999    0.316176633376      
  1 fxyy 
            0.352999    0.316176633376      
  1 fyyz 
            0.352999    0.316176633376      
  1 fxzz 
            0.352999    0.316176633376      
  1 fyzz 
            0.352999    0.316176633376      
  1 fxyz 
            0.352999    0.547633993174      
H2    15    5
  5 s    
               33.87    0.0607179345994     
               5.095    0.109507299103      
               1.159    0.161468449229      
              0.3258    0.154871066809      
              0.1027    0.049575118933      
  1 s    
              0.3258    0.307343053831      
  1 s    
              0.1027    0.129296844175      
  1 px   
               1.407    2.18427698453       
  1 py   
               1.407    2.18427698453       
  1 pz   
               1.407    2.18427698453       
  1 px   
               0.388    0.436495473997      
  1 py   
               0.388    0.436495473997      
  1 pz   
               0.388    0.436495473997      
  1 dxx  
               1.057    1.81359656262       
  1 dyy  
               1.057    1.81359656262       
  1 dzz  
               1.057    1.81359656262       
  1 dxy  
               1.057    3.14124139089       
  1 dxz  
               1.057    3.14124139089       
  1 dyz  
               1.057    3.14124139089       
H3    15    5
  5 s    
               33.87    0.0607179345994     
               5.095    0.109507299103      
               1.159    0.161468449229      
              0.3258    0.154871066809      
              0.1027    0.049575118933      
  1 s    
              0.3258    0.307343053831      
  1 s    
              0.1027    0.129296844175      
  1 px   
               1.407    2.18427698453       
  1 py   
               1.407    2.18427698453       
  1 pz   
               1.407    2.18427698453       
  1 px   
               0.388    0.436495473997      
  1 py   
               0.388    0.436495473997      
  1 pz   
               0.388    0.436495473997      
  1 dxx  
               1.057    1.81359656262       
  1 dyy  
               1.057    1.81359656262       
  1 dzz  
               1.057    1.81359656262       
  1 dxy  
               1.057    3.14124139089       
  1 dxz  
               1.057    3.14124139089       
  1 dyz  
               1.057    3.14124139089       
&
&wavefunction
  0.658567506               -0.205512276                0.0154329379             
  0                          0                          0.253112066              
  0                          0                         -0.0539076744             
  0                          0                         -0.0539796163             
  0.0181569384               0.0614409731               0.0273322695             
  0                          0                          0                        
 -0.00627625267              0.0494795714               0.0118863905             
  0                          0                          0                        
  0                          0                         -0.00453858652            
  0                         -0.00488974898              0                        
  0.0172789152               0                          0                        
  0                          0.444084776               -0.0446243786             
 -0.0878387271               0                         -0.00585274251            
 -0.00317659639              0                         -0.0152675123             
 -0.00675093255             -0.00122912036              0.00170331422            
 -0.000385101481             0                          0                        
  0.00121602702              0.444084776               -0.0446243786             
 -0.0878387271               0                          0.00585274251            
 -0.00317659639              0                          0.0152675123             
 -0.00675093255             -0.00122912036              0.00170331422            
 -0.000385101481             0                          0                        
 -0.00121602702            

  0                          1.29299765e-11             0                        
  0                          0.686079662                0                        
  0                         -0.162491491                0                        
  0                         -0.13763592                 0                        
  0                          0                          0                        
  0                          0                          0.0559717896             
  0                          0                          0                        
  0                          0                          0.0580139915             
  0                          0.0141593442               0                        
 -0.0102165235               0                          0                        
  0                          0                          0.00860770651            
  0                          0.572976948               -0.064459314              
 -0.0677012928               0                         -0.00546659647            
 -0.0038743952               0                         -0.00965549684            
 -0.00855513574              0.000749938213             0.00120325824            
  0.00142729579              0                          0                        
  0.00101607941             -0.572976948                0.064459314              
  0.0677012928               0                         -0.00546659647            
  0.0038743952               0                         -0.00965549684            
  0.00855513574             -0.000749938213            -0.00120325824            
 -0.00142729579              0                          0                        
  0.00101607941            

 -0.584879802               -0.253321037                0.0268053267             
  0                          0                          0.841801584              
  0                          0                         -0.132712698              
  0                          0                         -0.0658536929             
 -0.0148305292               0.123019674                0.000453015083           
  0                          0                          0                        
 -0.0195722564               0.061096938               -0.011326911              
  0                          0                          0                        
  0                          0                          0.000572291349           
  0                          0.00368544171              0                        
  0.00520182755              0                          0                        
  0                          0.266736793               -0.0379324351             
  0.0320251981               0                         -0.00185950978            
  0.000861501544             0                         -0.00415715376            
  0.00728666473              0.00135119966              0.00215262285            
  0.000496526771             0                          0                        
 -0.000977875169             0.266736793               -0.0379324351             
  0.0320251981               0                          0.00185950978            
  0.000861501544             0                          0.00415715376            
  0.00728666473              0.00135119966              0.00215262285            
  0.000496526771             0                          0                        
  0.000977875169           

  0                          0                          0                        
  0.999113686                0                          0                        
 -0.0126127907               0                          0                        
 -0.00165890396              0                          0                        
  0                          0                          0                        
  0                         -0.0518512832               0                        
  0                          0                          0                        
  0                         -0.0224263645               0                        
  0.00660746029              0                          0                        
  0                          0                         -0.0100227459             
  0                          0.00251104477              0                        
  0                          0                          0                        
  0                          0.00420263154              0                        
  0                          0.0145175183               0                        
  0                          0                          0                        
  0                         -0.00186140858             -0.0010831199             
  0                          0                          0                        
  0                          0.00420263154              0                        
  0                          0.0145175183               0                        
  0                          0                          0                        
  0                          0.00186140858             -0.0010831199             
  0                        

Alpha Occupation
1   1   1   1   

Beta Occupation
1   1   0   0   

CI Coeffs
  1                        

&

&Jastrow
ParticleTypes:	Electron_up	Electron_down
CorrelationFunctionType:	Cambridge2
NumberOfParameterTypes:	2
NumberOfParametersOfEachType:	1	8
Parameters:	0.1147925618	1.04453925101	7.88041772247	-55.008086367	195.848364925	-288.422951091	109.860870708	127.447788062	-94.2322483794
NumberOfConstantTypes:	2
NumberOfConstantsOfEachType:	1	1
Constants:	0.5	3

ParticleTypes:	Electron_up	Electron_up
CorrelationFunctionType:	Cambridge2
NumberOfParameterTypes:	2
NumberOfParametersOfEachType:	1	8
Parameters:	0.106868828676	0.751724101774	6.74371781933	-49.4666249546	168.099215131	-253.693770473	95.6799170297	150.089213831	-124.924993731
NumberOfConstantTypes:	2
NumberOfConstantsOfEachType:	1	1
Constants:	0.25	3

ParticleTypes:	Electron_up	Si
CorrelationFunctionType:	Cambridge2
NumberOfParameterTypes:	2
NumberOfParametersOfEachType:	1	8
Parameters:	0.0816482611315	-2.79574333504	6.67802213866	-113.680212032	394.440070397	-513.366994452	-217.060843347	94.3033951297	135.7932579
NumberOfConstantTypes:	2
NumberOfConstantsOfEachType:	1	1
Constants:	0	3

ParticleTypes:	Electron_up	H
CorrelationFunctionType:	Cambridge2
NumberOfParameterTypes:	2
NumberOfParametersOfEachType:	1	8
Parameters:	0.152990012168	-0.312076864177	4.6957711134	-33.5298366062	53.1556980948	42.9229233642	-106.926299628	-36.8925020965	84.2290279281
NumberOfConstantTypes:	2
NumberOfConstantsOfEachType:	1	1
Constants:	0	3

&Jastrow


