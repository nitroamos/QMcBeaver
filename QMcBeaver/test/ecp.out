QMcBeaver version 20080716 was compiled in double precision.
Compiled on Aug 24 2008 20:01:57
Compiler version 3.4.6 20060404 (Red Hat 3.4.6-8)
With the libraries: MPI BLAS LAPACK
Signaling: CHANNEL1 = 23 CHANNEL2 = 40


On atom 0: replacing 2 electrons with the C--ECP ecp using 14 grid points.
On atom 1: replacing 2 electrons with the C--ECP ecp using 14 grid points.
EupEdn:
Cambridge 2 particle jastrow parameters:
  g = 0.5  C = 3  L = 0.102677 (not optimized)
x = r / 9.739295
(x - 1)^3 (0.8310776 -2.376415 x +10.81917 x^2 -70.78149 x^3 +318.809 x^4 -665.4284 x^5 +478.9923 x^6 +178.9714 x^7 -272.3404 x^8 )

EupEup:
Cambridge 2 particle jastrow parameters:
  g = 0.25  C = 3  L = 0.121414 (not optimized)
x = r / 8.236286
(x - 1)^3 (0.5531108 -0.399739 x -2.278753 x^2 +24.76937 x^3 -72.89754 x^4 +72.3379 x^5 +23.17803 x^6 -59.30421 x^7 +15.33461 x^8 )

EupNuclear(C):
Cambridge 2 particle jastrow parameters:
  g = 0  C = 3  L = 0.1407478 (not optimized)
x = r / 7.104904
(x - 1)^3 (-1.398416 -4.195247 x +2.581357 x^2 +9.217973 x^3 -33.88012 x^4 -45.86722 x^5 +146.2064 x^6 -58.09619 x^7 -48.65388 x^8 )

EupNuclear(H):
Cambridge 2 particle jastrow parameters:
  g = 0  C = 3  L = 0.2097653 (not optimized)
x = r / 4.767232
(x - 1)^3 (-0.1524882 -0.4574645 x -4.083057 x^2 +57.53152 x^3 -273.1152 x^4 +509.8465 x^5 -258.7918 x^6 -265.778 x^7 +242.6451 x^8 )

EupEdnNuclear(C):
 Total Parameters = 64
 Free Parameters = 27
Cambridge 3 particle jastrow parameters:
  Nen = 4  Nee = 4  C = 3  L = 0.1143837 (not optimized)
x1  = r1  / 8.742507
x2  = r2  / 8.742507
x12 = r12 / 8.742507
(x1 - 1)^3 (x2 - 1)^3 (
+0.01087018                              +0.03261054 (x1 + x2)                    +0.09783163 x1 x2                        +73.18379  x1 x2 x12                     
+14.41264  (x1^2 + x2^2)                 -154.2881  (x1^2 x2 + x2^2 x1)           +745.4215  x1^2 x2^2                     -36.59189  (x1^2 + x2^2) x12             
-142.9358  (x1^2 x2 + x2^2 x1) x12       +544.9916  x1^2 x2^2 x12                 -24.68343  x12^2                         +50.29189  (x1 + x2) x12^2               
+241.0524  x1 x2 x12^2                   +103.1535  (x1^2 + x2^2) x12^2           -951.2456  (x1^2 x2 + x2^2 x1) x12^2     +717.527   x1^2 x2^2 x12^2               
-42.07295  (x1^3 + x2^3)                 +404.0114  (x1^3 x2 + x2^3 x1)           -1123.472  (x1^3 x2^2 + x2^3 x1^2)       +1161.96   x1^3 x2^3                     
+142.9358  (x1^3 + x2^3) x12             -272.4958  (x1^3 x2 + x2^3 x1) x12       -372.0759  (x1^3 + x2^3) x12^2           +660.7282  (x1^3 x2 + x2^3 x1) x12^2     
+813.144   (x1^3 x2^2 + x2^3 x1^2) x12^2 +532.5655  x1^3 x2^3 x12^2               +79.87872  x12^3                         -347.6107  (x1 + x2) x12^3               
+919.1774  x1 x2 x12^3                   +458.8342  (x1^2 + x2^2) x12^3           -400.4534  (x1^2 x2 + x2^2 x1) x12^3     -294.7634  x1^2 x2^2 x12^3               
-30.29492  (x1^3 + x2^3) x12^3           -90.88476  (x1^3 x2 + x2^3 x1) x12^3     -857.568   (x1^3 x2^2 + x2^3 x1^2) x12^3 +393.8196  x1^3 x2^3 x12^3               )

***************  TheMan.run();
 iteration            Eavg            Estd    Num. Samples           Tcorr      Acceptance         <p dR2>     Sample Var.         Summary
       -90    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -80    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -70    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -60    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -50    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -40    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -30    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -20    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
       -10    0.0000000000   0.0000000e+00               0   -2.0000000000         0.00000         0.00000    0.0000000000         0.00000 
         0  -14.3427994498   1.5475888e-01             100    6.9539436527      0.9801(56)      0.3457(63)    0.3409692445      -14.34(15) 
       110  -14.3541041099   1.4235792e-01             110    7.0584284741      0.9816(51)      0.3458(59)    0.3129548919      -14.35(14) 
       120  -14.3759063314   1.2410740e-01             120    6.2278049007      0.9826(47)      0.3477(55)    0.2943115518      -14.38(12) 
       130  -14.3728407019   1.1609313e-01             130    6.3127342097      0.9824(45)      0.3499(53)    0.2754135282      -14.37(12) 
       140  -14.3556072601   1.0946402e-01             140    6.3357278383      0.9771(69)      0.3488(52)    0.2628821307      -14.36(11) 
       150  -14.3091057695   1.0689036e-01             150    6.1110130797         0.97749      0.3499(49)    0.2785801507      -14.31(11) 
       160  -14.2391575058   1.3613166e-01             160    8.7620445871      0.9769(78)      0.3508(48)    0.3362869008      -14.24(14) 
       170  -14.1994020304   1.3613166e-01             170    9.0462471499      0.9781(78)      0.3502(46)    0.3462075552      -14.20(14) 
       180  -14.1559256145   1.4062491e-01             180    9.8115774256         0.97897      0.3488(44)    0.3607768678      -14.16(14) 
       190  -14.1148580984   1.4062491e-01             190    9.9997705072         0.97952      0.3488(43)    0.3737629588      -14.11(14) 
       200  -13.7850233759               0             100   -1.0000000000      0.9800(82)      0.3539(57)    0.3132685403       -13.78502 
       210  -13.6954066801               0             110   -1.0000000000         0.97875      0.3510(55)    0.3727282506       -13.69541 
       220  -13.6286343179               0             120   -1.0000000000      0.9802(64)      0.3507(53)    0.3922657120       -13.62863 
       230  -13.5545671213               0             130   -1.0000000000      0.9814(60)      0.3529(51)    0.4286441435       -13.55457 
       240  -13.5023956229               0             140   -1.0000000000         0.98233      0.3516(49)    0.4357868021       -13.50240 
       250  -13.4516998181               0             150   -1.0000000000      0.9834(57)      0.3534(47)    0.4488068501       -13.45170 
       260  -13.3968762066               0             160   -1.0000000000      0.9842(58)      0.3511(45)    0.4670692690       -13.39688 
       270  -13.3086771280               0             170   -1.0000000000      0.9849(58)      0.3513(44)    0.5705520992       -13.30868 
       280  -13.1866808971               0             180   -1.0000000000      0.9829(50)      0.3496(43)    0.8193344129       -13.18668 
       290  -13.1087655597               0             190   -1.0000000000         0.98331      0.3510(43)    0.8933544375       -13.10877 
       300  -13.0499090300               0             200   -1.0000000000         0.98385      0.3520(41)    0.9171225959       -13.04991 
       310  -12.9625164787               0             210   -1.0000000000      0.9841(43)      0.3521(40)    1.0292425106       -12.96252 
       320  -12.9032228874               0             220   -1.0000000000         0.98453      0.3513(40)    1.0626214237       -12.90322 
       330  -12.8960325128               0             230   -1.0000000000      0.9845(40)      0.3513(39)    1.0201248642       -12.89603 
       340  -12.9130952736               0             240   -1.0000000000      0.9850(37)      0.3518(38)    0.9877466952       -12.91310 
       350  -12.9282063929               0             250   -1.0000000000         0.98542      0.3524(37)    0.9546843621       -12.92821 
       360  -12.9534092861               0             260   -1.0000000000      0.9855(35)      0.3516(36)    0.9359870931       -12.95341 
       370  -12.9769527177               0             270   -1.0000000000         0.98564      0.3516(35)    0.9167894675       -12.97695 
       380  -12.9718736008               0             280   -1.0000000000         0.98596      0.3516(34)    0.8900260590       -12.97187 
       390  -12.9602801030               0             290   -1.0000000000         0.98630      0.3526(33)    0.8651290566       -12.96028 
       400  -12.9656596296               0             300   -1.0000000000         0.98659      0.3532(33)    0.8390500148       -12.96566 
       410  -12.9773884841               0             310   -1.0000000000         0.98682      0.3525(32)    0.8164416703       -12.97739 
       420  -12.9795514718               0             320   -1.0000000000      0.9869(35)      0.3526(31)    0.7922815100       -12.97955 
       430  -12.9712701963               0             330   -1.0000000000      0.9870(35)      0.3536(32)    0.7712101436       -12.97127 
       440  -12.9739432622               0             340   -1.0000000000      0.9872(35)      0.3541(31)    0.7492027797       -12.97394 
       450  -12.9856822371               0             350   -1.0000000000      0.9868(35)      0.3543(32)    0.7330623498       -12.98568 
       460  -13.0030585047               0             360   -1.0000000000      0.9864(26)      0.3533(31)    0.7245168646       -13.00306 
       470  -13.0073044766               0             370   -1.0000000000      0.9863(32)      0.3525(31)    0.7065917029       -13.00730 
       480  -13.0090485352               0             380   -1.0000000000      0.9861(32)      0.3527(31)    0.6891516369       -13.00905 
       490  -13.0185255305               0             390   -1.0000000000      0.9855(31)      0.3512(31)    0.6763719749       -13.01853 
       500  -13.0374823713               0             400   -1.0000000000      0.9853(31)      0.3516(30)    0.6737607983       -13.03748 
       500  -13.0374823713               0             400   -1.0000000000      0.9853(31)      0.3516(30)    0.6737607983       -13.03748 
***************  Print Results (ok = 1)
**************** Start of Results *****************

------------------- Energy -------------------
    -13.037482371312 +/- 99.000000000000      (400 samples, Tcorr   -1.00000)
  DeCorr_depth       samples           Ave           Std        StdStd           Var        VarStd    Corr. Len.      Variance          Skew      Kurtosis
0                        400    -13.037482  4.109287e-02  1.454672e-03  1.688624e-03  1.195533e-04      0.000000      0.673761      0.609834      0.832868
1                        200    -13.037482  5.764576e-02  2.889521e-03  3.323033e-03  3.331372e-04      1.967895      0.661284      0.614039      0.825214
2                        100    -13.037482  8.100015e-02  5.756430e-03  6.561024e-03  9.325433e-04      3.885427      0.649541      0.610103      0.786667
3                         50    -13.037482  1.126897e-01  1.138338e-02  1.269896e-02  2.565577e-03      7.520303      0.622249      0.504108      0.426884
4                         25    -13.037482  1.559312e-01  2.250673e-02  2.431454e-02  7.019003e-03     14.399028      0.583549      0.379429     -0.044399
5                         12    -13.011730  2.235730e-01  4.766592e-02  4.998488e-02  2.131362e-02     29.600961      0.549834      0.372508      0.176509
6                          6    -13.011730  2.669360e-01  8.441256e-02  7.125481e-02  4.506550e-02     42.196978      0.356274     -0.246656     -0.713717
7                          3    -13.011730  3.702382e-01  1.851191e-01  1.370764e-01  1.370764e-01     81.176388      0.274153      0.000000      0.000000
8                          1    -12.939382

----------------- Energy^2 -------------------
    170.649707380532 +/- 99.000000000000      (400 samples, Tcorr   -1.00000)

--------------- Kinetic Energy ---------------
     10.524158613154 +/- 0.847450895670       (400 samples, Tcorr   21.85781)

-------------- Potential Energy --------------
    -23.561640984466 +/- 0.792169891490       (400 samples, Tcorr   21.06536)

-------------- Nuc-Elec Energy ---------------
    -64.717155039947 +/- 0.902972719535       (400 samples, Tcorr   22.78962)

-------------- Elec-Elec Energy --------------
     21.589913123001 +/- 99.000000000000      (400 samples, Tcorr   -1.00000)

------------ AcceptanceProbability -----------
      0.985252051968 +/- 0.003145922628       (400 samples, Tcorr    3.22667)

------------ DistanceMovedAccepted -----------
      0.351611414057 +/- 0.003032237685       (400 samples, Tcorr    0.00000)

------------- DistanceMovedTrial -------------
      0.359024291583 +/- 0.002881310327       (400 samples, Tcorr    0.00000)

----------------- Walker Age -----------------
      0.000000000000 +/- 1.000000000000e+300  (0 samples, Tcorr   -2.00000)
dt_effective: 0.00979 +/- 0.00115

--------------- Virial Ratio (-<V>/<T>) ---------------
      0: 2.23881 +/- 0.19536
         Total Energy Estimator (KE/Virial): -10.52416 +/- 0.84745
         Total Energy Estimator (PE/Virial): -11.78082 +/- 0.39608

**************** End of Results *****************
***************  Finalize
Average iterations per hour:                              5.7548e+05
Average iterations*walkers per hour:                      1.151e+06
Average microseconds per sample:                          5987.1
Average microseconds per sample per num initial walkers:  2993.6

Wallclock Time:                         3127 ms   =                3.127 s    =             0.052117 mins =           0.00086861 hrs 
Total Time:                             3127 ms   =                3.127 s    =             0.052117 mins =           0.00086861 hrs     100.00 %
Initialization Time:                       7 ms   =                0.007 s    =           0.00011667 mins =           1.9444e-06 hrs       0.24 %
Equilibration Time:                      669 ms   =                0.669 s    =              0.01115 mins =           0.00018583 hrs      21.39 %
Propagation Time:                       2394 ms   =                2.394 s    =               0.0399 mins =             0.000665 hrs      76.57 %
Send Command Time:                         0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %
Gather Properties Time:                    0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.01 %
Synchronization Time:                      0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %
Poll for Command Time:                     0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %
Optimization Time:                         0 ms   =                    0 s    =                    0 mins =                    0 hrs       0.00 %

      Evaluate basis functions: 0 ( avg =     120.06 us, n=            987)      3.89 % cumulative =       3.89 %
         Matrix Multiplication: 0 ( avg =      26.63 us, n=            987)      0.86 % cumulative =       4.75 %
             Derivative Ratios: 0 ( avg =       8.53 us, n=            987)      0.28 % cumulative =       5.03 %
      Evaluate basis functions: 0 ( avg =     118.40 us, n=            987)      3.84 % cumulative =       8.86 %
         Matrix Multiplication: 0 ( avg =      25.04 us, n=            987)      0.81 % cumulative =       9.67 %
             Derivative Ratios: 0 ( avg =       6.26 us, n=            987)      0.20 % cumulative =       9.88 %
              Potential Energy: 0 ( avg =    2436.13 us, n=            987)     78.91 % cumulative =      88.79 %
                    EN Jastrow: 0 ( avg =      20.73 us, n=            987)      0.67 % cumulative =      89.46 %
                    EE Jastrow: 0 ( avg =      21.03 us, n=            987)      0.68 % cumulative =      90.14 %
                   EEN Jastrow: 0 ( avg =     234.94 us, n=            987)      7.61 % cumulative =      97.75 %
                Psi Quantities: 0 ( avg =       9.39 us, n=            987)      0.30 % cumulative =      98.05 %
            Walker Propagation: 0 ( avg =      49.91 us, n=            985)      1.62 % cumulative =      99.67 %
           Correlated Sampling: 0 ( avg =       0.00 us, n=            -15)      0.00 % cumulative =      99.67 %
         Calculate Observables: 0 ( avg =       7.99 us, n=            985)      0.26 % cumulative =      99.93 %
                   Entire Step: 0 ( avg =    3087.19 us, n=            985)    100.00 % cumulative =     100.00 %
Wallclock Time:                         3146 ms   =                3.146 s    =             0.052433 mins =           0.00087389 hrs 
